src/core/dev/hw_queue_tx.cpp

```diff
@@ -2013,6 +2018,8 @@ bool hw_queue_tx::expand_doca_task_pool(bool is_lso)
         PRINT_DOCA_ERR(hwqtx_logerr, rc, "DOCA expand task pool (LSO=%d) failed", is_lso);
         return false;
     }
+
+    m_task_list_expand_count += DOCA_EXPAND_BATCH_SIZE;
```

Comment:
See my comment about naming below. In this case, we better call it something like m_task_list_size

Reply:
maybe we can even fetch that number from doca to avoid keeping a variable to track it.

Reply:
Thanks. looked and no API unfortunately. as per your name comment - fixed.